debugging_overflow_on_images.md

```diff
@@ -0,0 +1,42 @@
+# Introduction
+
+Starting with Chrome 106, the `overflow` property is applied on the following elements: img, video and canvas. In earlier versions of Chrome, this property was ignored on these elements.
+
+This implies that an image which was earlier clipped to its content box can now draw outside those bounds. Common examples of CSS which can result in this are:
+
+```css
+img {
+  /* object-fit causes the image to scale and fill the complete box.
+     If the aspect ratio is different, the image will draw outside. */
+  object-fit: cover;
+  overflow: visible;
+}
+```
+
+For example, if your page has the following CSS:
```

Comment:
Isn't this two different examples? (The one above and the one below.)

Reply:
Yup, my bad.  That's why the starting line was "Common examples". These 2 were the breakages from the sites I inspected so explicitly mentioning both to help with debugging.